### PR TITLE
Add MySQL default credentials

### PR DESCRIPTION
**Pull request by gadnis**
Please correct me if I'm wrong. I try to find those credentials, but with no luck. I can't connect using these credentials using Sequel Pro aswell, so again, please write default credentials for users like me. Please save some time for me and other new comers. Thank you in advance. 😁

### CODE REVIEW

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md [hunk 1]
```diff
@@ -55,10 +55,23 @@ prestashop-git | -- Installation successful! --
 prestashop-git | \n* Almost ! Starting web server now\n
 ```
 
-At this point, your PrestaShop installation is ready and the website is available at http://localhost:8001.
+At this point, your PrestaShop installation is ready and the website is available at http://localhost:8001. 
 
 The default credentials for the back-office are `demo@prestashop.com` / `prestashop_demo`.
 
+Default MySql credentials to connect using 3rd party programs like Sequel Pro and others:
```
Comment (PierreRambaud) [inline src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]:
```suggestion
Default MySQL credentials to connect using 3rd party programs like Sequel Pro and others:
```

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md [hunk 2]
```diff
@@ -55,10 +55,23 @@ prestashop-git | -- Installation successful! --
...
+Username: `root`
```
Comment (PierreRambaud) [inline src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]:
```suggestion
Username: `prestashop`
```

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md [hunk 3]
```diff
@@ -55,10 +55,23 @@ prestashop-git | -- Installation successful! --
...
+Password: `prestashop`
+Port: `32768`
+
+You can check MySql port using command line
```
Comment (PierreRambaud) [inline src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]:
```suggestion
You can check MySQL port using command line
```

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md [hunk 4]
```diff
@@ -55,10 +55,23 @@ prestashop-git | -- Installation successful! --
...
 
-At this point, your PrestaShop installation is ready and the website is available at http://localhost:8001.
+At this point, your PrestaShop installation is ready and the website is available at http://localhost:8001. 
 
 The default credentials for the back-office are `demo@prestashop.com` / `prestashop_demo`.
 
+Default MySql credentials to connect using 3rd party programs like Sequel Pro and others:
+Username: `root`
+Password: `prestashop`
+Port: `32768`
```
Comment (PierreRambaud) [inline src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]:
```suggestion
Port: `3306`
```
as defined here: https://github.com/PrestaShop/PrestaShop/blob/develop/docker-compose.yml#L12 :thinking:

Comment (gadnis) [reply src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]: For me Port 3306 doesn’t work. 
I had to use ‘docker ps’ to check port. And it was written 32768->3306

Comment (PierreRambaud) [reply src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]: The port is maybe random :thinking:

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md (hunk 2 above)
Comment (gadnis) [reply src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]: > 

For me prestashop username not working. I'm using the latest repository.

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md (hunk 4 above)
Comment (PierreRambaud) [reply src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]: I confirm this port is generated dynamically, I'm not having the same one

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md (hunk 2 above)
Comment (PierreRambaud) [reply src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]: I'm using the latest version I'm able to connect with the `prestashop` user :thinking:

Comment (PierreRambaud) [reply src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]: I checked again, and you're right, the prestashop user isn't able to connect.

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md [hunk 5]
```diff
@@ -55,10 +55,23 @@ prestashop-git | -- Installation successful! --
...
 
-At this point, your PrestaShop installation is ready and the website is available at http://localhost:8001.
+At this point, your PrestaShop installation is ready and the website is available at http://localhost:8001. 
 
 The default credentials for the back-office are `demo@prestashop.com` / `prestashop_demo`.
 
+Default MySql credentials to connect using 3rd party programs like Sequel Pro and others:
+Username: `root`
+Password: `prestashop`
+Port: `3306`
```
Comment (PierreRambaud) [inline src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]: The port is generated dynamically, 
```suggestion
```

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md [hunk 6]
```diff
@@ -55,10 +55,23 @@ prestashop-git | -- Installation successful! --
...
+
+You can check MySQL port using command line
+
+```
+docker ps
```
Comment (PierreRambaud) [inline src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]:
```suggestion
docker-compose ps mysql
```

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md [hunk 7]
```diff
@@ -55,10 +55,23 @@ prestashop-git | -- Installation successful! --
...
+```
+
+Now you can see all docker containers.
```
Comment (PierreRambaud) [inline src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]:
```suggestion
```

**Comment by gadnis** [comment]
Thanks @PierreRambaud ;)

**Comment by PierreRambaud** [comment]
One more review and merge please :)

**Comment by gadnis** [comment]
> One more review and merge please :)

Sorry @PierreRambaud , you are telling me that I can merge it? 
I'm not seeing merge option down here.

Screen Shot 2020-01-17 at 15 41 12

**Comment by matks** [comment]
> > One more review and merge please :)
> 
> Sorry @PierreRambaud , you are telling me that I can merge it?
> I'm not seeing merge option down here.
> 
> Screen Shot 2020-01-17 at 15 41 12

No, it was a message for other maintainers ^^ basically he wants this topic to require 2 approvals

**Comment by matks** [comment]
Thank you @gadnis

**Comment by gadnis** [comment]
> Thank you @gadnis

Yey! My first contribution approved! 😄 Thanks!

**Pull request merged**